src/widget/widget-config.js

```diff
@@ -0,0 +1,55 @@
+
+// Retrieves widget config based on the script tag url.
+
+export const WIDGET_BASE_URL = `${process.env.WIDGET_ORIGIN}/widget.js`
+// The ".js" extension is omitted so the HTTP request can bypass
+// the wordpress web server.
+export const REVERSE_PROXY_WIDGET_PATH = '/arc-widget'
+
+export function isWidgetUrl (url) {
+    const { href } = new URL(url)
+    return  href.startsWith(WIDGET_BASE_URL)
+}
+
+function getConf (urlObj, conf = {}) {
+    const [_, queryStr] = urlObj.href.split(/#|[?]/)
+    const searchParams = new URLSearchParams(queryStr)
+    conf.clientKey = searchParams.get('target')
```

Comment:
Where do you see `target`? It says `integration` on the installation instructions on https://portal.saturn.tech/

Reply:
don't have access to portal (login doesn't work for me). Thought it was "target" for some reason. changed this